lua/terrortown/menus/gamemode/guide/equipment.lua

```diff
@@ -0,0 +1,10 @@
+--- @ignore
+
+CLGAMEMODESUBMENU.base = "base_gamemodesubmenu"
+
+CLGAMEMODESUBMENU.priority = 98
+CLGAMEMODESUBMENU.title = "submenu_guide_equipment_title"
+
+function CLGAMEMODESUBMENU:Populate(parent)
```

Comment:
Empty function

Reply:
yes, because there are no misc settings as of yet. Same as it was before this move